src/test/resources/properties/duplicate.properties

```diff
@@ -0,0 +1,2 @@
+BaseUrl_QA:Internal=https://internal.test.com
+BaseUrl_QA:External=https://external.test.com
```

Comment:
remember about new line here and below

Reply:
you want a carriage return at the end of each of the test property files? if so no problem

Reply:
yes - as github points this out

Reply:
ah fair enough...good to know 👍

Reply:
and the final kind request - squash all commits into one and make sure that the commit message is short and clear (like the title of this PR) :)